### PR TITLE
Mexican road shields.

### PR DESCRIPTION
**Pull request by zerebubuth**
Added handling of Mexican road shields.

In addition to national `MEX` roads, there are road networks in most states (although I couldn't find examples in Edomex or Tlaxcala), plus interior and exterior ring roads in Mexico City (`CDMX`). Sometimes these are in relations, but mostly not.

Connects to #1491.

### CODE REVIEW

File: integration-test/1491-mexican-shields.py [hunk 1]
```diff
@@ -0,0 +1,1074 @@
...
+                    "highway": "motorway",
+                },
+            ),
+        )
+
+        self.assert_has_feature(
+            z, x, y, "roads", {
+                "id": 285597347,
+                "shield_text": "15D",
+                "network": "MX:MEX",
```
Comment (nvkelso) [inline integration-test/1491-mexican-shields.py]: Should this be `MX:MX` to follow 2-char convention elsewhere?

Comment (nvkelso) [reply integration-test/1491-mexican-shields.py]: Ah, ISO uses 3-char for subdivision codes for Mexican states so using `MEX` here would be more consisent with that. 👍

File: integration-test/1491-mexican-shields.py [hunk 2]
```diff
@@ -0,0 +1,1074 @@
...
+                    "ref": "Q. ROO 5",
+                    "highway": "secondary",
+                },
+            ),
+        )
+
+        self.assert_has_feature(
+            z, x, y, "roads", {
+                "id": 399152055,
+                "shield_text": "5",
```
Comment (nvkelso) [inline integration-test/1491-mexican-shields.py]: That's an epic transform!

File: integration-test/1491-mexican-shields.py [hunk 3]
```diff
@@ -0,0 +1,1074 @@
...
+                'alt_name': 'Circuito Bicentenario', 'oneway': 'yes',
+                'foot': 'no', 'ref': 'INT', 'highway': 'motorway',
+            }),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 123899735,
+                'shield_text': type(None),
+                'network': 'MX:CMX:INT',
```
Comment (nvkelso) [inline integration-test/1491-mexican-shields.py]: Are there any of these that are just `MX:CMX` to test for?

Comment (zerebubuth) [reply integration-test/1491-mexican-shields.py]: I couldn't find any. Mexico City seems to have a bunch of different `ref`s, e.g: [3 NTE](https://www.openstreetmap.org/way/332816679), [Eje Central](https://www.openstreetmap.org/way/24706165), [Periférico](https://www.openstreetmap.org/way/182927016). Unfortunately, I wasn't able to find any more information about them on the internet, such as whether they're any kind of official designation of road network. For the moment, I've left them out - please follow up with an issue if they ought to be `MX:CMX` networks.

**Pull request merged**